cypress/integration/lock-exceptions/user-adds-lock-exception.js

```diff
@@ -53,24 +53,21 @@ Then('organisation units are fetched', () => {
 })
 
 When('the user selects an organisation unit', () => {
-    cy.getWithDataTest('{add-lock-exception-modal-org-unit-tree}')
-        .findByRole('button', { name: 'Bo' })
-        .find('input[type="checkbox"]')
-        .click()
+    cy.getWithDataTest('{add-lock-exception-org-unit-tree}')
+        .findByRole('checkbox', { name: 'Bo' })
+        .click({ force: true })
 })
 
 When('the user selects a period', () => {
     cy.getWithDataTest('{period-picker-option-year}').click()
-    cy.findByRole('menuitem', { name: '2014' }).click()
+    cy.contains('2014').click()
```

Comment:
The [`PeriodPicker`'s `material-ui` dropdown component was replaced with `SingleSelectField` from `@dhis2/ui`](https://github.com/dhis2/data-administration-app/blob/f2a16ecaa4aa9538c3d30eb07763e5d0ae29453d/src/pages/add-lock-exception/AddLockExceptionForm/PeriodPicker.js#L267), which is missing `role` attributes.